### PR TITLE
CMR-7311

### PR DESCRIPTION
**Pull request by ygliuvt**
As a CMR developer, I would like to set up a lambda function to consume the CMR collection indexing events to populate graph db.

### CODE REVIEW

File: graph-db/indexer/handler.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports.index = async event => {
```
Comment (ygliuvt) [inline graph-db/indexer/handler.js:3]: This function will be completely replaced with the actual indexing code that is coming when we work on CMR-7307. So ignore what's happening inside here.

File: graph-db/indexer/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "dependencies": {
+    "serverless": "^2.43.0"
+  },
+  "name": "graph-lambda",
```
Comment (mschmele) [inline graph-db/indexer/package.json:5]: I'd name this something more descriptive like `cmr-graphdb-indexer`

**Review by mschmele — COMMENTED** [review]
We should either keep jest/eslint configs at the top-level directory, or copy them to each project. It may be worth it to do that in the next ticket, though. Your code additions look good to me, but I'm not sure how we actually want to organize this project

File: graph-db/indexer/handler.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  console.log(`Ryan, Sarah and Mark will see ingest event: [${JSON.stringify(event, null, 2)}].`)
```
Comment (jaybarra) [inline graph-db/indexer/handler.js:4]: Should remove personal identifiers

File: graph-db/indexer/handler.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  return {
+    statusCode: 200,
+    body: JSON.stringify(
+      {
+        message: 'Processed ingest event successfully!',
+        input: event,
```
Comment (jaybarra) [inline graph-db/indexer/handler.js:10]: What is the purpose of returning the event? If you are returning more than a 200 should that jut be the request body?

**Review by jaybarra — CHANGES_REQUESTED** [review]

File: graph-db/indexer/handler.js (hunk 4 above)
Comment (mschmele) [reply graph-db/indexer/handler.js:10]: I'm going to be modifying this, so it shouldn't actually matter what is here

File: graph-db/indexer/handler.js (hunk 3 above)
Comment (ygliuvt) [reply graph-db/indexer/handler.js:4]: See my comment: `This function will be completely replaced with the actual indexing code that is coming when we work on CMR-7307. So ignore what's happening inside here.`

File: graph-db/indexer/handler.js (hunk 4 above)
Comment (ygliuvt) [reply graph-db/indexer/handler.js:10]: See my comment: This function will be completely replaced with the actual indexing code that is coming when we work on CMR-7307. So ignore what's happening inside here.

**Review by mschmele — APPROVED** [review]
There will be some more moving around that's needed in the next PR for common util functions, but this PR satisfies the ticket - looks good to me

**Review by jaybarra — APPROVED** [review]

**Pull request merged**